### PR TITLE
[NO CHANGELOG] Upgrade biome design-tokens to 0.4.4 and react to 0.27.2

### PR DESCRIPTION
**Pull request by jiyounglee**
Upgrade biome design-tokens to 0.4.4 and react to 0.27.2

### CODE REVIEW

File: packages/checkout/widgets-lib/src/views/top-up/TopUpMenuItem.tsx [hunk 1]
```diff
@@ -25,15 +27,19 @@ export function TopUpMenuItem({
...
+          />
+        ) : (
+          <MenuItem.Icon icon={icon} />
+        )}
         <MenuItem.Label size="medium">
           {heading}
         </MenuItem.Label>
         <MenuItem.IntentIcon
-          icon={intentIcon}
+          icon={intentIcon as AllSingleVariantIconKeys}
```
Comment (glomotion) [inline packages/checkout/widgets-lib/src/views/top-up/TopUpMenuItem.tsx:42]: so it kinda depends on how many icons we want to support in here. If we want to support Dual and Single icon variants properly, then we should maybe do as above and use the typeGuard: `isDualVariantIcon()`

**Review by jhesgodi — DISMISSED** [review]

**Review by zaidarain1 — DISMISSED** [review]

File: packages/passport/sdk-sample-app/src/components/Message.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ function Message() {
   return (
     <CardStack title="Message">
       <Sticker
```
Comment (haydenfowler) [inline packages/passport/sdk-sample-app/src/components/Message.tsx:19]: @glomotion should this version upgrade have any impact on the Sticker component?
Left is after this version bump, right is prior

image

Comment (glomotion) [reply packages/passport/sdk-sample-app/src/components/Message.tsx:19]: @haydenfowler so stickers have not changed, but Sticker.FramedIcon has. 😅 Think we just need one extra prop here. Ill fix.

Comment (glomotion) [reply packages/passport/sdk-sample-app/src/components/Message.tsx:19]: OK this commit should fix it: https://github.com/immutable/ts-immutable-sdk/pull/2219/commits/819e41e162858f81a93b6d3f55860ba1e4b9bdda

Comment (haydenfowler) [reply packages/passport/sdk-sample-app/src/components/Message.tsx:19]: @glomotion Same result unfortunately 😅

Comment (glomotion) [reply packages/passport/sdk-sample-app/src/components/Message.tsx:19]: @haydenfowler OK should definitely be fixed this time. Sorry for the false start. 🤪

**Review by haydenfowler — APPROVED** [review]

**Review by zaidarain1 — APPROVED** [review]

**Review by jhesgodi — APPROVED** [review]

**Pull request merged**